samples/snippets/visualbasic/VS_Snippets_Misc/tpl_parallel/vb/simpleforeach.vb

```diff
@@ -1,37 +1,83 @@
 ﻿'<snippet03>
-Imports System.IO
-Imports System.Threading
-Imports System.Threading.Tasks
-Imports System.Drawing
+Imports System.Collections.Concurrent
```

Comment:
Our build system now expects a *.vbproj to verify that this code compiles. Could you please add a new project file here:

*samples/snippets/csharp/VS_Snippets_Misc/tpl_parallel/vb/simpleforeach.vbproj*

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>net5.0</TargetFramework>
  </PropertyGroup>

</Project>
```